docs/configurations.md

```diff
@@ -26,6 +26,7 @@ This list contains configuration variables that can be used with all balena devi
 | BALENA_HOST_FIREWALL_MODE | string | false | off | Toggle firewall modes between on, off, and auto. | v11.9.1 |
 | BALENA_HOST_DISCOVERABILITY | boolean | false | true | Enable / Disable Avahi to run which will allow the device to respond to requests such as network scans. | v11.9.2 |
 | BALENA_HOST_SPLASH_IMAGE | integer | true | /boot/splash/balena-logo-default.png | Allows changing splash screen on boot to user defined file from userspace. | v12.3.0 |
+| BALENA_SUPERVISOR_DISABLE_HARDWARE_METRICS | boolean | false | false | Disables hardware metrics such as CPU temperature and memory usage from being reported to the backend and displayed in the dashboard. Useful for devices with bandwidth sensitivity. | v12.8.0 |
```

Comment:
Added version (v12.8.0)

Reply:
Agreed that this might be a problem in the future, the PR might have to be added separately after the feature is merged to avoid this dev flow